docs/docs/vertical-tab-w-line.md

```diff
@@ -0,0 +1,114 @@
+---
+id: vertical-tabs-w-line
+title: Vertical Tabs With Line
+sidebar_label: Vertical Tabs With Line
+---
+
+``` html live
+<>
+<div class="lil-flex"> 
+    <div class="lil-w-2/5 sm:lil-hidden">
+        <ul class="lil-vertical-tabs">
+            <li class="lil-vertical-tab lil-vertical-tab-active" onclick="verticalTabsWLine(event,'tab1')">
+                <a class="lil-vertical-tab-container">
+                    <div class="lil-vertical-tab-line"></div>
```

Comment:
y cant we left border for tab ? 🤔

Reply:
Tried this, actually there are two left borders of grey and blue for active tab, the blue one is not of full height so we have a separate element of border height

Reply:
there is also a part where there is no bottom border, hence it's more straight forward using another element, tried using margins to simulate this but didn't work properly, if I but the bottom border for the parent we won't be able to have that space in between, and if bottom border is for current element then the height of left border becomes full

Reply:
![Screenshot from 2020-11-10 10-52-56](https://user-images.githubusercontent.com/70994472/98631224-fb8bf800-2342-11eb-9eb0-9b7a27c153bc.png)

Reply:
> there is also a part where there is no bottom border
👀